recipe/meta.yaml

```diff
@@ -107,13 +104,11 @@ requirements:
   run:
   - armadillo==8.200.0
   - blas==1.1
-  - boost==1.67
```

Comment:
Why is this removed? Xalan and Doxygen are not needed because they are for building docs.

Reply:
I copied the dependencies directly from the environment.yml dependencies section.  Boost was not in environment.yml.